velox/functions/prestosql/tests/ArithmeticTest.cpp

```diff
@@ -699,5 +699,32 @@ TEST_F(ArithmeticTest, truncate) {
   EXPECT_DOUBLE_EQ(truncate(123456789012345678901.23, -21).value(), 0.0);
 }
 
+TEST_F(ArithmeticTest, betaCDF) {
```

Comment:
Perhaps, create a new test for probability functions: ProbabilityTest.cpp

Reply:
Created new testfile